config/deb.am

```diff
@@ -85,3 +88,16 @@ deb-utils: deb-local rpm-utils-initramfs
 	    $$pkg8 $$pkg9 $$pkg10 $$pkg11;
 
 deb: deb-kmod deb-dkms deb-utils
+
+copy-deb:
+	if ! test -d debian; then cp -r contrib/debian debian; chmod +x debian/rules; fi;
```

Comment:
Will something like this work? And depend on the debian target rather than copy-deb:

```suggestion
debian:
	cp -r contrib/debian debian; chmod +x debian/rules
```